archetypes/helidon/src/main/archetype/mp/custom/custom-mp.xml

```diff
@@ -31,12 +31,6 @@
     <exec src="/mp/common/common-mp.xml"/>
     <source src="/mp/custom/observability.xml"/>
     <output>
-        <files if="${security}">
```

Comment:
What's the issue with this block ? It's not a `.mustache` neither does the path contain `__pkg__`.

Reply:
If `security` is selected but does not contain `http-signature`, it results in copying the whole directory for some reason. Looks like a issue in build-tools. I did not investigate yet. I with create a unit test and check the behavior.

Other reason to remove this block is that in `common/security.xml, the file `keystore.p12` is already included so it does not make sense to have this logic in `custom-mp.xml`.